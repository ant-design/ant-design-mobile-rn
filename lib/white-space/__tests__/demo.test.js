import rnDemoTest from '../../../tests/shared/demoTest';

rnDemoTest('white-space');
