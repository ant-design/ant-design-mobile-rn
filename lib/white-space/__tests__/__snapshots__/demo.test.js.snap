// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/white-space/demo/basic.tsx correctly 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "height": 3,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 30,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#bbb",
        }
      }
    >
      Block
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 6,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 30,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#bbb",
        }
      }
    >
      Block
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 9,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 30,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#bbb",
        }
      }
    >
      Block
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 15,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 30,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#bbb",
        }
      }
    >
      Block
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 21,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "height": 30,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#bbb",
        }
      }
    >
      Block
    </Text>
  </View>
</View>
`;
